app/metrics.yaml

```diff
@@ -539,3 +539,35 @@ custom_tab:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-03-01"
+
+activation:
+  identifier:
+    type: string
+    lifetime: ping
+    description: >
+      An hashed and salted version of the Google Advertising ID from the device.
```

Comment:
Please also include here that this is never sent with the client id. Also document the case where this is the client id.

Reply:
I've addressed the first request. We are no longer sending the `client_id`, not even when the GAID is not available. In that case, we're generating a random UUID (the `activation_id`), which will be stable throughout the life of the application, and *exclusively* sent with this ping.